cloud-watch-to-slack-testing/deployment/index.js

```diff
@@ -258,6 +258,10 @@ function ecsTaskStateChangeMessageText(message) {
   const taskArn = message?.detail?.taskArn;
   const lastStatus = message?.detail?.lastStatus;
   let messageText = `Task ${taskArn} is now ${lastStatus}`;
+  const taskContainers = message?.detail?.containers
+    ?.map((container) => container.name)
+    .join(", ");
```

Comment:
To be robust when the `containers` field is missing, probably needs a `?.join` so that an `undefined` continues to propagate. 

Also, right now, if the value of `containers` is a scalar (which it probably never will be, but you never know), this code will throw at the `map` invocation.  What happens when the lambda throws, are we notified?  If so, probably not worth worrying about.

Reply:
I don't believe we are notified. The lambda does have a dead queue configured but I don't think we do anything with it https://github.com/dockstore/dockstore-deploy/blob/38977e218b16a73d0a29167d00f3c89bbdb09b3d/cloudwatch-to-slack.yml.template#L146-L147

Reply:
Created https://ucsc-cgl.atlassian.net/browse/SEAB-6856